samples/bot-task-module/csharp/Helper/DeeplinkHelper.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.